#include "exception.hpp"

int main()
{

  return 0;
}
